src/views/api_v1.js

```diff
@@ -70,17 +69,17 @@ var Api = {
 		return userID + "_" + global._socket_counter++;
 	},
 
-	getUserID: function (req) {
-		if (!req.user) {
+	getUserID: function (res) {
```

Comment:
https://github.com/oauthjs/express-oauth-server/blob/master/test/integration/index_test.js#L96 that is how express-oauth-server does this.